### PR TITLE
version bump to 0.9.0

### PR DESCRIPTION
**Pull request by ericmjl**

### CODE REVIEW

**Comment by ericmjl** [comment]
ping @jakirkham @ocefpaf; building for PyMC3 (also don't see a PR open for version 0.9.0, so I thought I'd contribute it in.)

ping @twiecki @fonnesbeck FYI only.

**Comment by ocefpaf** [comment]
@ericmjl consider adding yourself as a maintainer.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -7,7 +7,7 @@ package:
 source:
   fn: Theano-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/T/Theano/Theano-{{ version }}.tar.gz
-  md5: f2d0dfe7df141115201077cd933b2c52
+  md5: 2df1b85fe634b7d9241721b2966e6ae2
```
Comment (jakirkham) [inline recipe/meta.yaml]: Could we please use `sha256` from [pypi.org]( https://pypi.org/ ) instead?

Comment (ericmjl) [reply recipe/meta.yaml]: Updated in new push!

**Comment by jakirkham** [comment]
Also the `install` requirements have changed.

```yaml
- numpy >=1.9.1
- scipy >=0.14
- six >=1.9.0
```

Would also like to add the following simple things since we are updating this anyways.

* Drop all `build` dependencies other than `python` and `setuptools`.
* Add `license_file: LICENSE.txt` to the `about` section.

**Comment by ericmjl** [comment]
@jakirkham just want to confirm, the `install` requirements are under the `run` sub-section or `build` sub-section?

**Comment by jakirkham** [comment]
Under `run` sorry.

**Comment by ericmjl** [comment]
@jakirkham done!

**Comment by jakirkham** [comment]
Thanks.

**Comment by jakirkham** [comment]
I think we need `libgpuarray` as well. Fortunately I think it is ready to go. Just got lost in the shuffle.

cc @abergeron

xref: https://github.com/conda-forge/staged-recipes/issues/2411
xref: https://github.com/conda-forge/staged-recipes/pull/2457

**Comment by ericmjl** [comment]
@jakirkham added!

**Comment by ericmjl** [comment]
@ocefpaf @jakirkham yay! It passed on CircleCI. Just waiting for the rest to finish up. Once it's merged, I'll put in a re-PR for PyMC3.

**Comment by jakirkham** [comment]
As [noted]( https://github.com/conda-forge/theano-feedstock/pull/10#issuecomment-289039892 ), I think we need `libgpuarray` as a run time dependency.

**Comment by ericmjl** [comment]
@jakirkham already done.

**Comment by jakirkham** [comment]
Missed that. Sorry. Thanks for adding it.

**Comment by ericmjl** [comment]
haha, I'm surprised now, it's AppVeyor's turn to be back-logged? Travis used to be the culprit lol.

**Comment by ericmjl** [comment]
ping @jakirkham: not sure why AppVeyor failed - 'curl' error?

**Comment by ocefpaf** [comment]
@ericmjl you need to re-render with latest conda-smithy to fix it.

**Comment by ericmjl** [comment]
@ocefpaf done!

**Comment by jakirkham** [comment]
Restarted CIs so that we can test on the recently add `libgpuarray` on conda-forge.

**Comment by ericmjl** [comment]
Got it!

**Comment by ericmjl** [comment]
@jakirkham @ocefpaf is there by any chance some notification system for when the CIs pass? (Just wondering, no pressure!)

**Comment by jakirkham** [comment]
Probably by email, but I already struggle to manage the notifications from this org without the CIs. Feel free to just ping when it passes and one of us can merge.

**Comment by ericmjl** [comment]
@jakirkham @ocefpaf I was actually wondering if there's a way I could be informed! 😄 That way I can know when to ping you guys.

**Comment by jakirkham** [comment]
There are, but not easily and doing them selectively is not simple or ideal. Best thing I can recommend is look at the AppVeyor queue and guesstimate when it will be done and check back then.

**Comment by ericmjl** [comment]
ping @jakirkham

**Comment by jakirkham** [comment]
Thanks @ericmjl.

**Pull request merged**